### ISSUE TITLE
Comparing ECC and RSA keys

### ISSUE THREAD

**Issue by Akuli** [Legrandin/pycryptodome#659]
```python3
>>> from Crypto.PublicKey import ECC, RSA
>>> ECC.generate(curve='p384') == RSA.generate(bits=4096)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/a/foo/env/lib/python3.10/site-packages/Crypto/PublicKey/ECC.py", line 807, in __eq__
    return other.pointQ == self.pointQ
AttributeError: 'RsaKey' object has no attribute 'pointQ'
```

Presumably `EccKey`'s `__eq__` should do an `isinstance()` check, and return `False` (or even better, `NotImplemented`) when the argument is not an `EccKey` object.

**Comment by Legrandin** [comment]
Fixed with https://github.com/Legrandin/pycryptodome/commit/1681c6b3b3a51af8d2276b76856eb450519fd0e5

**Comment by Legrandin** [comment]
Fixed in v3.17
